data/mods/Magiclysm/Spells/attunements/Cleansing_Flame.json

```
@@ -75,5 +75,55 @@
     "min_range": 4,
     "max_range": 8,
     "range_increment": 0.12
+  },
+  {
+    "id": "cleansingflame_hermes_heal",
+    "type": "SPELL",
+    "spell_class": "CLEANSING_FLAME",
+    "name": "Hermes Reborn",
+    "description": "In the sea without lees\nStandeth the bird of Hermes\nEating his wings variable\nAnd maketh himself yet full stable",
```

```suggestion
    "description": {
      "ctxt": "This spell heals the caster, but then causes area-effect damage including to the caster, then does a 'Pain Split' to distribute the damage among the caster's limbs.",
      "str": "In the sea without lees\nStandeth the bird of Hermes\nEating his wings variable\nAnd maketh himself yet full stable",
    },
```
This may well not be properly-formatted JSON, but the intent is to give the translators some hint of how to translate this. @BrettDong, any comments?

---

Context should usually be a short string. `"//~"` is prefered for comments like this.